qemu/tests/block_copy.py

```diff
@@ -321,3 +322,40 @@ def clean(self):
         while self.trash_files:
             tmp_file = self.trash_files.pop()
             utils.system("rm -f %s" % tmp_file, ignore_status=True)
+
+    def get_md5sum(self, create=False):
+        """
+        Get md5sum value of files.
+        :param create: create a new file if True
+        :return: md5sum value
+        """
+        params = self.params
+        session = self.vm.wait_for_login(
```

Comment:
here is idea that you can use md5sum filename > filename.md5 to recording it in guest filesystem, then verify it by call cmd md5sm check filename.md5, so that you can skip recording, checking steps in your python script.